maps/away/ships/dominia/dominian_science_vessel/dominian_science_vessel.dm

```diff
@@ -0,0 +1,146 @@
+/datum/map_template/ruin/away_site/dominian_science_vessel
+	name = "Dominian Science Vessel"
+	description = "Based on the Lammergeier-class corvette, this vessel has been repurposed by House Volvalaad for long range survey and scientific tasks. Due to its repurposement, the vessel features an enlarged hangar and shuttle, as well as scientific labs and a smaller defensive armament."
+	suffixes = list("ships/dominia/dominian_science_vessel/dominian_science_vessel.dmm")
+	sectors = list(ALL_TAU_CETI_SECTORS, ALL_COALITION_SECTORS, SECTOR_BADLANDS)
```

Comment:
Why does a Dominian ship spawn in the Coalition, a nation which has extremely poor relations with Dominia? Also, if you are going to keep it in the Coalition, make sure to use sectors_blacklist to prevent it from spawning in places like Haneunim/Burzsia.